src/orion/algo/space.py

```diff
@@ -901,7 +915,12 @@ def cardinality(self):
 
     def get_prior_string(self):
         """Build the string corresponding to current prior"""
-        return "fidelity({}, {}, {})".format(self.low, self.high, self.base)
+        args = [str(self.low), str(self.high)]
+
+        if self.base != 2:
```

Comment:
Same thing here, is the base is set explicitly to 2 by the user, it will not be added to the prior string.

Reply:
yes, that is expected